src/components/PlantsV2/index.tsx

```diff
@@ -94,16 +78,16 @@ export default function PlantsDashboardV2(): JSX.Element {
     <>
       {sectionHeader(strings.MORTALITY_RATE)}
       <Grid item xs={isMobile ? 12 : 3}>
-        <TotalMortalityRateCard observation={latestObservation} />
+        <TotalMortalityRateCard plantingSiteId={selectedPlantingSiteId} />
       </Grid>
       <Grid item xs={isMobile ? 12 : 3}>
-        <HighestAndLowestMortalityRateZonesCard observation={latestObservation} />
+        <HighestAndLowestMortalityRateZonesCard plantingSiteId={selectedPlantingSiteId} />
       </Grid>
       <Grid item xs={isMobile ? 12 : 3}>
-        <HighestAndLowestMortalityRateSpeciesCard observation={latestObservation} />
+        <HighestAndLowestMortalityRateSpeciesCard plantingSiteId={selectedPlantingSiteId} />
       </Grid>
       <Grid item xs={isMobile ? 12 : 3}>
-        <LiveDeadPlantsPerSpeciesCard observation={latestObservation} />
+        <LiveDeadPlantsPerSpeciesCard plantingSiteId={selectedPlantingSiteId} />
```

Comment:
i'm on the fence here.. maybe these smaller widgets are ok receiving the data as props?

Reply:
Hm, yeah it's kind of a tossup. I guess I liked the consistency of selecting the observation result data from state using a selector at the point of use. However, it does create a dependency between the index component and each of the cards, wherein the index is responsible for dispatching the appropriate thunks. Rather than the data flow being explicit in the props interface.

Reply:
ideally these smaller widgets should be 'story-able', props would make that easier me thinks.. like you said .. 6 of one, half a dozen of the other

Reply:
Okay, most of these cards have other data that they get with selectors internally. I think that sorta makes them all less than story-able without rewriting most of them. So given that, do you think consistency wins out here?

Reply:
Got it, yes if we are already using selectors within those components.. this is the right path 👍 
Thanks for sharing that extra context.

Reply:
Cool, thanks for the discussion on this. Gives me a better sense of when to use props vs selectors in the future :)